obspy/clients/nrl/tests/test_nrl.py

```diff
@@ -35,7 +35,6 @@ def test_nrl_types(self):
             self.assertIsInstance(nrl, NRL)
         self.assertIsInstance(self.nrl_local, LocalNRL)
         self.assertIsInstance(self.nrl_online, RemoteNRL)
-        self.assertIsInstance(self.nrl_default, RemoteNRL)
```

Comment:
This tests if NRL() is called with no args returns a RemoteNRL. I think this is a good test.